fetch.go

```diff
@@ -213,12 +273,28 @@ func (p *Purge) FetchVpcs() {
 		}
 
 		region := *svc.Config.Region
+		filters := p.filters.Vpc
 
 		vpcs := make([]*ec2.Vpc, 0)
 		for _, vpc := range resp.Vpcs {
 			// don't delete default vpc
-			if !*vpc.IsDefault {
+			if *vpc.IsDefault {
```

Comment:
you may want to use `aws.BoolValue` and in general use `aws.*Value` functions for accessing aws fields.